include/robot_localization/common_utilities.h

```diff
@@ -0,0 +1,78 @@
+#ifndef ROBOT_LOCALIZATION_COMMON_UTILITIES_H
```

Comment:
This file seems to be pretty much just about debugging. I wonder if you should just make this file DebugLogger.h and leave the macro here. Thoughts?